staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1734,6 +1785,11 @@ type VolumeProjection struct {
 	// serviceAccountToken is information about the serviceAccountToken data to project
 	// +optional
 	ServiceAccountToken *ServiceAccountTokenProjection `json:"serviceAccountToken,omitempty" protobuf:"bytes,4,opt,name=serviceAccountToken"`
+	// clusterTrustBundlePEM requests projection of a ClusterTrustBundle's
+	// pemTrustAnchors field into a file.
```

Comment:
Done.

Reply:
Following up, why should we drop the field when the feature gate is disabled rather than throwing an error?  A workload coded to use a cluster trust bundle projection will be broken if we silently drop the field.